src/application/wizard/service.test.ts

```diff
@@ -101,4 +167,40 @@ describe('saveDetailsPage', () => {
             ],
         });
     });
+
+    it('should throw when userId is not owner', async () => {
```

Comment:
Is this a repetition of the test at line 40 : "should throw if submission ownership is invalid"